docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,485 @@
+    )
+
+    track_artists = (
+        pl.scan_csv(self.DATASETS_BASE + "r_track_artist.csv")
+        .select(pl.col("track_id"), pl.col("artist_id"))
+        .sort("track_id")
+        .collect()
+    )
+
+    audio_features = (
+        pl.scan_csv(self.DATASETS_BASE + "audio_features.csv")
+        .select(pl.col("id").alias("track_id"), "danceability", "energy", "tempo",)
+        .filter(pl.col("danceability").is_between(*self.DANCEABILITY_RANGE))
+        .filter(pl.col("energy").is_between(*self.ENERGY_RANGE))
+        .filter(pl.col("tempo").is_between(*self.TEMPO_RANGE))
+        .sort("track_id")
+        .collect()
+    )
+
+    danceability_tracks = (
+        track_artists.join(artists, on="artist_id")
+        .join(tracks, on="track_id")
+        .join(audio_features, on="track_id")
+    )
+
+    self.provider.io.info(f"{self.name} calculating final results...")
+
+    matches = pl.DataFrame()
+    for row in sample.rows(named=True):
+        artist = row["artist_name"]
+        track = row["track_name"]
+        m = (
+            danceability_tracks.select(
+                "artist_name",
+                "track_name",
+                "danceability",
+                "energy",
+                ("https://open.spotify.com/track/" + pl.col("track_id")),
+            )
+            .filter(pl.col("artist_name") == artist)
+            .filter(pl.col("track_name") == track)
+            .limit(1)
+        )
+
+        if not m.is_empty():
+            matches = pl.concat([matches, m])
+
+    matches = matches.sort("danceability", descending=True)
+
+    self.provider.write_to_anchor(
+        "Output", pa.Table.from_pandas(matches.to_pandas())
+    )
+
+    self.provider.io.info(f"{self.name} finished.")
+```
+
+# Package Into a YXI
+Now that we've created our code, it's time to package it all up into a portable YXI archive. For this, we will be using the `ayx_plugin_cli create-yxi` command. The process looks like the following:
+
+```bash
+~/MyWorkspace$ ayx_plugin_cli create-yxi
+[Creating YXI] started
+[Creating YXI] -- generate_config_files:generate_config_xml
+[Creating YXI] -- generate_config_files:generate_tool_config_xml
+[Creating YXI] .  generate_config_files:generate_manifest_jsons
+[Creating YXI] Generating manifest.json for DanceableLyrics...
+[Creating YXI] Done!
+...omitted...
+~\MyWorkspace\main.pyz -e ayx_python_sdk.providers.amp_provider.__main__:main
+[Creating YXI] Created shiv artifact at: ~\MyWorkspace\main.pyz
+[Creating YXI] .  create_yxi:create_yxi
+[Creating YXI] finished
+```
+
+# Run the Test Client
+Before proceeding on to running within Designer, it's good practice to do some basic testing with `ayx-sdk-cli`'s `plugin run` command to check for any errors. From your workspace:
+
+```bash
+ayx-sdk-cli plugin run DanceableLyrics -o danceable.csv
+```
+
+The above command will run our plugin and produce `danceable.csv`. You may run `ayx-sdk-cli plugin run --help` for additional options.
```

Comment:
TW review:

The above command runs our plugin and produces `danceable.csv`. You can run `ayx-sdk-cli plugin run --help` for additional options.